index.js

```diff
@@ -44,6 +44,10 @@ var CqlshDriver = Base.extend({
     metaData.push('PRIMARY KEY ' + primaryKey);
     cql.push(metaData.join(', '));
     cql.push(')');
+    if (constraints.compression) {
+      cql.push('WITH compression =');
+      cql.push(constraints.compression);
+    }
```

Comment:
You mean the commit `ee6c71b540cf28b13b6dbff5815f563068afac93`, right? I think the pull request that you refer to is part of this here pull request, already. So if you merge the other pull request first, and refresh this one, the above (changed) lines should not show up anymore. Does that make sense?